libraries/csharp-net22-buildtime/context/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+dotnet restore
```

Comment:
what's restore do? this will be executing in a pretty much clean environment every time, so not sure if that matters

Reply:
it downloads the packages from nuget - it's kind of a crappy name but it's like `pip install -r requirements.txt`